Hashiconf docs changes (#2178)
hashicorp/boundary: website/content/docs/release-notes/v0_9_0.mdx

---
layout: docs
page_title: v0.9.0
description: |-
  Boundary release notes for v0.9.0
---

# [Boundary v0.9.0](https://www.boundaryproject.io/downloads)

The release notes below contain information about new functionality available in the Boundary v0.9.0 release.

> To see a granular record of when each item was merged into the Boundary project, please refer to the [Changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md).

Reviewer: It's not clear what is being suggested here by "a granular record of when each item was merged". The changelog has no information about timing for the various features -- is the idea that you'd then click on the PR links and look at the dates/times?

Author: pinging @xingluw for comments on the release notes

> To learn about what Boundary consists of, we highly recommend you start at the [Getting Started Page](/docs/getting-started).

Reviewer: This reads a bit awkwardly to me, maybe:

Reviewer: ```suggestion
To learn about Boundary generally, we highly recommend you start at the [Getting Started Page](/docs/getting-started).
```

Lastly, for instructions on how to upgrade an existing Boundary deployment to v0.9.0, please review Boundary's [general upgrade guide](https://learn.hashicorp.com/tutorials/boundary/upgrade-version).

## Boundary v0.9.0 Highlights

**HCP Boundary Public Beta:** Boundary is coming to the HashiCorp Cloud Platform! HCP Boundary provides an easy way to securely access 
critical systems with fine-grained authorizations based on trusted identities. Boundary on HashiCorp Cloud Platform provides a fully managed, 

> single workflow to securely connect to hosts and critical systems across Kubernetes clusters, cloud service catalogs, and on-premises infrastructure.

Reviewer: Do we want to highlight Kubernetes here, considering we have no native integration yet and it'd have to be done via manual configuration via e.g. static?

You can now [try HCP Boundary](https://learn.hashicorp.com/collections/boundary/hcp-getting-started) for free during our [Public Beta](https://cloud.hashicorp.com/products/boundary).

**Self-Managed Workers:** With HCP Boundary, administrators have the option of setting up private, self-managed workers for infrastructure access. This provides the 
security of having privately managed workers while providing the low overhead of a managed service. Learn more about setting up self-managed workers
[here](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-workers).

> **Worker Registration Enhancements:** Administrators now have a second method of authenticating workers, and we call these PKI Workers (the original 

Reviewer: ```suggestion
**Worker Registration Enhancements:** Administrators now have a second method of authenticating workers, referred to as PKI Workers (the original 
```
